command_map.go

```
@@ -0,0 +1,54 @@
+package main
+
+import "inputs/login"
+import "github.com/tidwall/gjson"
+import "encoding/json"
+
+var commands = map[string]Command{
+	"login": login.Login{},
+}
+
+//Test JSON
+var jsonFromApiGateway = []string{`{"command":"login", "data":{"userName":"some@email.com","password":"mypassword"}}`,
+	`{"command":"register", "data":{"userName":"some@email.com","password":"mypassword"}}`}
+
+//This is just for testing json processing.  Will move to test later.
+func main() {
+	for i := 0; i < len(jsonFromApiGateway); i++ {
+		willBeHandle(jsonFromApiGateway[i])
+	}
+}
+
+//This will be set to the lambda handle
+func willBeHandle(stringJson string) {
+
+	commandName := gjson.Get(stringJson, "command").String()
+	if !validateCommand(commandName) {
+		println("Invalid command")
+		return
```

you should probably return an error here, return fmt.Errorf("Invalid command"), then print solely from main